playwright/drawer-panel-dropdown.spec.ts

```
@@ -0,0 +1,334 @@
+  test('actions dropdown shows all expected menu items', async ({ page }) => {
+    await drawerHelpers.openDrawer(page);
+    await drawerHelpers.waitForDrawerReady(page);
+
+    const actionsToggle = page.locator('#notifications-actions-toggle');
+    await actionsToggle.click();
+
+    const dropdown = page.locator('#notifications-actions-dropdown');
+    await expect(dropdown).toBeVisible({ timeout: 5000 });
+
+    // Bulk action items
+    await expect(dropdown.getByRole('menuitem', { name: /Mark selected .* as read/ })).toBeVisible();
```

ESLint

[error] Replace `dropdown.getByRole('menuitem',·{·name:·/Mark·selected·.*·as·read/·})` with `⏎······dropdown.getByRole('menuitem',·{·name:·/Mark·selected·.*·as·read/·})⏎····`

(prettier/prettier)